packages/react-native/Libraries/Components/TextInput/InputAccessoryView.js

```diff
@@ -86,7 +86,19 @@ type Props = $ReadOnly<{|
 |}>;
 
 class InputAccessoryView extends React.Component<Props> {
+  // once the 'visionos' is implemented as a platform, we can remove this
+  componentDidMount() {
+    if (Platform.isVisionOS) {
+      console.warn('InputAccessoryView is not available on visionOS platform.');
```

Comment:
based on feedback from RN Discord, this should probably use `warnOnce` to match what tvOS maintainer and Microsoft has done.